src/shared/state/Applet/Applet.thunk.ts

```diff
@@ -40,10 +40,10 @@ export const getAppletWithItems = createAsyncThunk(
 );
 
 export const createApplet = createAsyncThunk(
-  AppletThunkTypePrefix.Create,
-  async (body: SingleApplet, { rejectWithValue, signal }) => {
+  AppletThunkTypePrefix.Update,
```

Comment:
why did you change this:
```suggestion
  AppletThunkTypePrefix.Create,
```